### PR TITLE
doc: Update Changelog for the new release v0.9.3

### PR DESCRIPTION
**Pull request by finikorg**
Update Changelog in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,16 @@ Installation can be done in one of 2 options:
 
 
 ## Changelog
+### v0.9.3
+- tbtadm: added `peers` command and xdomain now shown in topology output
+- tbtadm: added `add` comand for adding to ACL without `approve`
```
Comment (ybernat) [inline README.md]: command

File: README.md [hunk 2]
```diff
@@ -88,6 +88,16 @@ Installation can be done in one of 2 options:
...
+- tbtadm: fixed adding to ACL in secure level 0
```
Comment (ybernat) [inline README.md]: security level

Comment (ybernat) [reply README.md]: 2. Fixes should be under `resolved issues` (or, if you prefer, merge the sections)

**Review by ybernat — CHANGES_REQUESTED** [review]
Thanks for handling this too!

File: README.md (hunk 2 above)
Comment (finikorg) [reply README.md]: @ybernat Thanks for review, I would prefer to squash them to Changelog for easy navigation, I will fix those issues and add squashing patch.
@westeri

File: README.md (hunk 1 above)
Comment (ybernat) [reply README.md]: ?

**Comment by finikorg** [comment]
@ybernat have you seen the latest code?

File: README.md [hunk 3]
```diff
@@ -88,19 +88,35 @@ Installation can be done in one of 2 options:
 
 
 ## Changelog
+### v0.9.3
+- tbtadm: added `peers` command and xdomain is now shown in the topology output
```
Comment (ybernat) [inline README.md]: What about the addition of `tbtxdomain` for auto-loading `tuhnderbolt-net` driver?

Comment (finikorg) [reply README.md]: Added line to changelog about autoload

**Pull request merged**